ghcjs-src/Miso/String.hs

```diff
@@ -45,11 +47,12 @@ instance FromJSON MisoString where
 -- | Convenience class for creating `MisoString` from other string-like types
 class ToMisoString str where
   toMisoString :: str -> MisoString
+  ms :: str -> MisoString
```

Comment:
What’s the difference between `toMisoString` and `ms`? Is `ms` just a shorter name? If so, should it really be a class method that instances can potentially override?